.github/workflows/CI.yml

```diff
@@ -27,7 +27,22 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install --upgrade platformio
+    - name: Install Dependencies
+      run: |
+        pio platform install espressif32 --with-package framework-arduinoespressif32 --with-package tool-esptoolpy
+        pio lib -e m5plusdemo install --no-save
```

Comment:
Just out of curiosity: is there a particular reason for choosing m5plusdemo for this step, over any of the others?

Reply:
I just chose m5plusdemo because it's the default env. I could create a new env and include the common, m5stick, and m5stick-c-plus deps, and use that instead. 

https://github.com/PlummersSoftwareLLC/NightDriverStrip/blob/cfe39e7b59c3ac99df923d363de3ac66563ab534/platformio.ini#L13

Reply:
Something like this:

```
[env:all_deps]
lib_deps      =  ${common.lib_deps}
                 ${m5stick-c.lib_deps}
                 ${m5stick-c-plus.lib_deps}
```

Reply:
I think it makes sense to pull in most if not all dependencies that get used more than once in the overall CI run. As does creating a separate environment for it; it's clear then what determines the dependency cache coverage and it can be maintained accordingly.

Reply:
added

Reply:
squashed the commits